modules/sql-engine/src/test/java/org/apache/ignite/internal/sql/engine/sql/SqlAlterColumnDdlParserTest.java

```diff
@@ -96,22 +101,42 @@ public void testDefault() {
      */
     @Test
     public void testSetDataType() {
-        validateDataType("SET DATA TYPE INTEGER", "INTEGER", null, null);
-        validateDataType("SET DATA TYPE INTEGER NOT NULL", "INTEGER", true, null);
-        validateDataType("SET DATA TYPE INTEGER NULL", "INTEGER", false, null);
-        validateDataType("SET DATA TYPE INTEGER DEFAULT -1", "INTEGER", null, -1L);
-        validateDataType("SET DATA TYPE INTEGER DEFAULT NULL", "INTEGER", null, null);
-        validateDataType("SET DATA TYPE INTEGER NOT NULL DEFAULT -1", "INTEGER", true, -1);
-        validateDataType("SET DATA TYPE INTEGER NULL DEFAULT NULL", "INTEGER", false, null);
+        IgniteSqlAlterColumn alterColumn = parseAlterColumn("ALTER TABLE t ALTER COLUMN c SET DATA TYPE INTEGER");
+        expectDataType(alterColumn, "INTEGER", null, null);
+        expectUnparsed(alterColumn, "ALTER TABLE \"T\" ALTER COLUMN \"C\" SET DATA TYPE INTEGER");
+
+        alterColumn = parseAlterColumn("ALTER TABLE t ALTER COLUMN c SET DATA TYPE INTEGER NOT NULL");
+        expectDataType(alterColumn, "INTEGER", true, null);
+        expectUnparsed(alterColumn, "ALTER TABLE \"T\" ALTER COLUMN \"C\" SET DATA TYPE INTEGER NOT NULL");
+
+        alterColumn = parseAlterColumn("ALTER TABLE t ALTER COLUMN c SET DATA TYPE INTEGER NULL");
+        expectDataType(alterColumn, "INTEGER", false, null);
+        expectUnparsed(alterColumn, "ALTER TABLE \"T\" ALTER COLUMN \"C\" SET DATA TYPE INTEGER NULL");
+
+        alterColumn = parseAlterColumn("ALTER TABLE t ALTER COLUMN c SET DATA TYPE INTEGER DEFAULT -1");
+        expectDataType(alterColumn, "INTEGER", null, -1);
+        expectUnparsed(alterColumn, "ALTER TABLE \"T\" ALTER COLUMN \"C\" SET DATA TYPE INTEGER DEFAULT -1");
+
+        alterColumn = parseAlterColumn("ALTER TABLE t ALTER COLUMN c SET DATA TYPE INTEGER NOT NULL DEFAULT -1");
```

Comment:
this case is duplicates below, I think the case `SET DATA TYPE INTEGER DEFAULT NULL` is missing

Reply:
Thanks. Fixed.